src/services/configurations/users.ts

```diff
@@ -7,7 +7,16 @@ export default (
   httpAuth: HttpInstance
 ): ConfigurationsUsersService => ({
   async get(userId, options) {
-    return (await client.get(httpAuth, `/users/${userId}`, options)).data;
+    const { data } = await client.get(httpAuth, `/users/${userId}`, {
+      ...options,
+      customResponseKeys: [
+        'data.*',
+        'staffConfiguration.*',
+        'patientConfiguration.*',
```

Comment:
I think this is wrong, if I look at the swagger this should be `staffConfiguration.data` and `patientConfiguration.data`